docs/querying-data.rst

```diff
@@ -0,0 +1,207 @@
+Querying data in Kingfisher Views
+=================================
+
+Summary tables in Kingfisher Views are generated from data in `Kingfisher Process <https://kingfisher-process.readthedocs.io/en/latest/>`__, which is organized by collection. Each collection contains either data downloaded from an OCDS publisher using a `Kingfisher Scrape <https://kingfisher-scrape.readthedocs.io/en/latest/>`__ spider, or loaded locally.
+
+A Kingfisher Views schema can contain summaries of one or more collections.
+
+OCDS data can be published using either `releases or records <https://standard.open-contracting.org/latest/en/getting_started/releases_and_records/>`__. Releases are point in time updates about a contracting process and a record provides an index of all the releases for a contracting process.
+
+Kingfisher Process automatically transforms OCDS releases to create compiled release collections,  containing a single compiled release per contracting process populated with the latest value of each field.
+
+Tables in Kingfisher Views can contain summaries of:
+
+* individual releases
+* compiled releases generated by Kingfisher Process
+* compiled releases published as part of an OCDS record
+
+Since most analysis is much easier to perform on compiled releases, we recommend working with compiled releases to begin with.
+
+List all schemas in the database
+--------------------------------
```

Comment:
This is already covered in https://kingfisher-views.readthedocs.io/en/latest/cli/use.html#list-views

Reply:
We can add a link to that, but this documentation is intended to serve users who only have access via Redash and the methods there don't work in that case.

Reply:
At least use the much shorter `\dn` below.

Reply:
`\dn` doesn't work in Redash:

![image](https://user-images.githubusercontent.com/19265814/77978719-454d4280-735f-11ea-96d5-0c6692faa774.png)

Reply:
Ah, too bad – I guess this is the way to do it then.